src/coreclr/clrdefinitions.cmake

```diff
@@ -148,6 +148,8 @@ if(FEATURE_MERGE_JIT_AND_ENGINE)
   add_definitions(-DFEATURE_MERGE_JIT_AND_ENGINE)
 endif(FEATURE_MERGE_JIT_AND_ENGINE)
 add_compile_definitions($<$<NOT:$<BOOL:$<TARGET_PROPERTY:CROSSGEN_COMPONENT>>>:FEATURE_MULTICOREJIT>)
+add_compile_definitions($<$<NOT:$<BOOL:$<TARGET_PROPERTY:CROSSGEN_COMPONENT>>>:FEATURE_MULTICOREJIT_NDIRECTSTUB>)
```

Comment:
Is there a reason to split this into different feature flags? I think this should be just unconditional part of MultiCoreJIT.

Reply:
I used it for test separately. I will fix it. Thank you.